cmake/test/src/test_feature.py

```diff
@@ -0,0 +1,107 @@
+####
+# test_basic.py:
+#
+# Basic CMake tests.
+#
+####
+import platform
+
+import cmake
+import settings
+
+
+FEATURE_BUILD = cmake.get_build(
```

Comment:
For the other test cases you set the return value to `_` and used the fixture instead.

Reply:
This is my personal opinion, so feel free to disagree, but I prefer to avoid side effects when including python files. Would it be possible to kick off the cmake build through some pytest init method instead?

Reply:
yes, good point.